src/main/java/FileHandler.java

```diff
@@ -0,0 +1,106 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class FileHandler {
+
+    public static ArrayList<String> Lines = new ArrayList<String>();
+
+    public static void initFiles(){
+        File dataDir = new File("./DataDir");
+        if (!dataDir.exists()) dataDir.mkdir();
+
+        try{
+            File dataFile = new File("./DataDir/Data.txt");
```

Comment:
It would be great to eliminate the use of magic strings, in this case, consider replacing this with a variable or static variable instead!